core/modules/parsers/wikiparser/rules/quoteblock.js

```diff
@@ -3,30 +3,7 @@ title: $:/core/modules/parsers/wikiparser/rules/quoteblock.js
 type: application/javascript
 module-type: wikirule
 
-Wiki text rule for quote blocks. For example:
-
-```
-	<<<.optionalClass(es) optional cited from
-	a quote
-	<<<
-	
-	<<<.optionalClass(es)
-	a quote
-	<<< optional cited from
-```
-
-Quotes can be quoted by putting more <s
-
-```
-	<<<
-	Quote Level 1
-	
-	<<<<
-	QuoteLevel 2
-	<<<<
-	
-	<<<
-```
```

Comment:
IMO this should not be removed. Just remove the indentation, then it should work, if copy/pasted to a tiddler.

Reply:
Thanks @pmario. @Jermolene Now that I have identified the bug as parsing tabs/whitespace & clarified the docs, should we keep or remove the example in the parser?